po/nl_NL.po

```diff
@@ -0,0 +1,590 @@
+"Last-Translator: Heimen Stoffels <vistausss@outlook.com>\n"
+"Language-Team: Dutch <vistausss@outlook.com>\n"
+"Language: nl_NL\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 2.1.1\n"
+
+#. / These keys must be valid since they will be used across the app
+#: /home/felipe/Code/Notes-up/po/../src/Application.vala:37
+msgid "<Ctrl>M"
+msgstr "<Ctrl>M"
+
+#: /home/felipe/Code/Notes-up/po/../src/Application.vala:38
+msgid "<Ctrl>S"
+msgstr "<Ctrl>S"
+
+#: /home/felipe/Code/Notes-up/po/../src/Application.vala:39
+msgid "<Ctrl>Q"
+msgstr "<Ctrl>Q"
+
+#: /home/felipe/Code/Notes-up/po/../src/Application.vala:40
+msgid "<Ctrl><Shift>N"
+msgstr "<Ctrl><Shift>N"
+
+#: /home/felipe/Code/Notes-up/po/../src/Application.vala:41
+msgid "<Ctrl>F"
+msgstr "<Ctrl>F"
+
+#: /home/felipe/Code/Notes-up/po/../src/Application.vala:42
+msgid "<Ctrl>K"
+msgstr "<Ctrl>K"
+
+#: /home/felipe/Code/Notes-up/po/../src/Application.vala:43
+msgid "<Ctrl>B"
+msgstr "<Ctrl>B"
+
+#: /home/felipe/Code/Notes-up/po/../src/Application.vala:44
+msgid "<Ctrl>I"
+msgstr "<Ctrl>I"
+
+#: /home/felipe/Code/Notes-up/po/../src/Application.vala:45
+msgid "<Ctrl>T"
+msgstr "<Ctrl>T"
+
+#: /home/felipe/Code/Notes-up/po/../src/Application.vala:46
+msgid "<Ctrl><Shift>I"
+msgstr "<Ctrl><Shift>I"
+
+#: /home/felipe/Code/Notes-up/po/../src/Application.vala:65
+msgid "Notes-Up"
+msgstr "Notes-Up"
+
+#: /home/felipe/Code/Notes-up/po/../src/Application.vala:66
+msgid "Your Markdown Notebook."
+msgstr "Jouw markdown-notitieboek."
+
+#: /home/felipe/Code/Notes-up/po/../src/Application.vala:67
+msgid "About Notes"
+msgstr "Over Notities"
+
+#: /home/felipe/Code/Notes-up/po/../src/Application.vala:108
+#: /home/felipe/Code/Notes-up/po/../src/Widgets/Headerbar.vala:257
+#: /home/felipe/Code/Notes-up/po/../src/Widgets/Sidebar/NotebookItem.vala:54
+#: /home/felipe/Code/Notes-up/po/../src/Widgets/Sidebar/NotebookList.vala:32
+msgid "New Notebook"
+msgstr "Nieuw notitieboek"
+
+#: /home/felipe/Code/Notes-up/po/../src/Plugins/Break.vala:28
+msgid "Page break when exporting to PDF: <break>"
+msgstr "Pagina-einde tijdens exporteren naar PDF: <break>"
+
+#: /home/felipe/Code/Notes-up/po/../src/Plugins/Break.vala:32
+msgid "Page break"
+msgstr "Pagina-einde"
+
+#: /home/felipe/Code/Notes-up/po/../src/Plugins/Break.vala:47
+msgid "Page break on export: <break>"
+msgstr "Pagina-einde tijdens exporteren: <break>"
+
+#: /home/felipe/Code/Notes-up/po/../src/Plugins/Color.vala:35
+msgid ""
+"Set font color for the line with <color [#color]> or for some text with "
+"<color [color] [text]>"
+msgstr ""
+"Stel de lettertypekleur in van de regel met <color [#kleur]> of, voor tekst, "
+"met <color [kleur] [tekst]>"
+
+#: /home/felipe/Code/Notes-up/po/../src/Plugins/Color.vala:39
+#: /home/felipe/Code/Notes-up/po/../src/Plugins/Color.vala:68
+msgid "Font color"
+msgstr "Lettertypekleur"
```

Comment:
Google translate says that this should be two words, is that correct? 

`"Lettertype Kleur"`

Reply:
No, that's incorrect. "Lettertypekleur" is one word.

Reply:
Thank you, just double checking :)

Reply:
Sure, no problem :)